crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -115,14 +120,19 @@ impl CallContractCommand {
 			full_message.push_str(&format!(" --gas {}", gas_limit));
 		}
 		if let Some(proof_size) = self.proof_size {
-			full_message.push_str(&format!(" --proof_size {}", proof_size));
+			full_message.push_str(&format!(" --proof-size {}", proof_size));
```

Comment:
No related to this feature but a very easy fix, display correctly the flags: `--proof-size`